Bottom: b659f113056f82a4af447874d41bb1f81ee55030
Top:    b857c6917ad1e7f3e08d704b2880a2a697545152
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-03 19:26:54 +0000

Temporarily undo a few find.c changes


---

diff --git a/src/find.c b/src/find.c
index d11f77d..05081b0 100644
--- a/src/find.c
+++ b/src/find.c
@@ -257,7 +257,7 @@ typedef struct
  * some local identifiers
  */
 static Coord Bloat = 0;
-//static void *thing_ptr1, *thing_ptr2, *thing_ptr3;
+static void *thing_ptr1, *thing_ptr2, *thing_ptr3;
 static int thing_type;
 static bool User = false;    /* user action causing this */
 static bool drc = false;     /* whether to stop if finding something not found */
@@ -3475,7 +3475,7 @@ drc_callback (DataType *data, LayerType *layer, PolygonType *polygon,
   PinType *pin = (PinType *) ptr2;
   PadType *pad = (PadType *) ptr2;
 
-  SetFubarThing (type, ptr1, ptr2, ptr2);
+  SetThing (type, ptr1, ptr2, ptr2);
 
   switch (type)
     {
@@ -3660,7 +3660,7 @@ DRCAll (void)
             SET_FLAG (SELECTEDFLAG, line);
             DrawLine (layer, line);
             drcerr_count++;
-            SetFubarThing (LINE_TYPE, layer, line, line);
+            SetThing (LINE_TYPE, layer, line, line);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
             violation = pcb_drc_violation_new (_("Line width is too thin"),
@@ -3704,7 +3704,7 @@ DRCAll (void)
             SET_FLAG (SELECTEDFLAG, arc);
             DrawArc (layer, arc);
             drcerr_count++;
-            SetFubarThing (ARC_TYPE, layer, arc, arc);
+            SetThing (ARC_TYPE, layer, arc, arc);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
             violation = pcb_drc_violation_new (_("Arc width is too thin"),
@@ -3749,7 +3749,7 @@ DRCAll (void)
             SET_FLAG (SELECTEDFLAG, pin);
             DrawPin (pin);
             drcerr_count++;
-            SetFubarThing (PIN_TYPE, element, pin, pin);
+            SetThing (PIN_TYPE, element, pin, pin);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
             violation = pcb_drc_violation_new (_("Pin annular ring too small"),
@@ -3781,7 +3781,7 @@ DRCAll (void)
             SET_FLAG (SELECTEDFLAG, pin);
             DrawPin (pin);
             drcerr_count++;
-            SetFubarThing (PIN_TYPE, element, pin, pin);
+            SetThing (PIN_TYPE, element, pin, pin);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
             violation = pcb_drc_violation_new (_("Pin drill size is too small"),
@@ -3824,7 +3824,7 @@ DRCAll (void)
             SET_FLAG (SELECTEDFLAG, pad);
             DrawPad (pad);
             drcerr_count++;
-            SetFubarThing (PAD_TYPE, element, pad, pad);
+            SetThing (PAD_TYPE, element, pad, pad);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
             violation = pcb_drc_violation_new (_("Pad is too thin"),
@@ -3869,7 +3869,7 @@ DRCAll (void)
             SET_FLAG (SELECTEDFLAG, via);
             DrawVia (via);
             drcerr_count++;
-            SetFubarThing (VIA_TYPE, via, via, via);
+            SetThing (VIA_TYPE, via, via, via);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
             violation = pcb_drc_violation_new (_("Via annular ring too small"),
@@ -3901,7 +3901,7 @@ DRCAll (void)
             SET_FLAG (SELECTEDFLAG, via);
             DrawVia (via);
             drcerr_count++;
-            SetFubarThing (VIA_TYPE, via, via, via);
+            SetThing (VIA_TYPE, via, via, via);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
             violation = pcb_drc_violation_new (_("Via drill size is too small"),
@@ -3944,7 +3944,7 @@ DRCAll (void)
             SET_FLAG (SELECTEDFLAG, line);
             DrawLine (layer, line);
             drcerr_count++;
-            SetFubarThing (LINE_TYPE, layer, line, line);
+            SetThing (LINE_TYPE, layer, line, line);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
             violation = pcb_drc_violation_new (_("Silk line is too thin"),
@@ -3995,7 +3995,7 @@ DRCAll (void)
             SET_FLAG (SELECTEDFLAG, element);
             DrawElement (element);
             drcerr_count++;
-            SetFubarThing (ELEMENT_TYPE, element, element, element);
+            SetThing (ELEMENT_TYPE, element, element, element);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
